### ISSUE TITLE
export_to_tensorboard fails: no attribute called FileWriter

### ISSUE THREAD

**Issue by dem108** [Azure/MachineLearningNotebooks#752]
The following example notebook fails when running `export_to_tensorboard(root_run, logdir)`:
https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/training-with-deep-learning/export-run-history-to-tensorboard/export-run-history-to-tensorboard.ipynb

The error message is as follows:
```
exportedTBlogs


---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-7-7f8972e77f18> in <module>
     12 
     13 # export run history for the project
---> 14 export_to_tensorboard(root_run, logdir)
     15 
     16 # or export a particular run

/anaconda/envs/azureml_py36/lib/python3.6/site-packages/azureml/tensorboard/export.py in export_to_tensorboard(run_data, logsdir, logger, recursive)
    102                 writer.close()
    103             # Create new file writer for each new runid
--> 104             writer = tf.summary.FileWriter(os.path.join(logsdir, run_id))
    105         metrics = run_metrics.cells
    106         step = 0

AttributeError: module 'tensorboard.summary._tf.summary' has no attribute 'FileWriter'
```

I'm not sure if this is caused by a bug from Azure ML SDK, or a compatibility issue of TensorFLow version. Did we need to specificy old version of TF?

**Comment by andrewkinsella** [comment]
Hello, can someone from microsoft be assigned to look at this please ?
This notebook is part of the training set and it's not working.

**Comment by swanderz** [comment]
@jpe316 ?

**Comment by andrewkinsella** [comment]
@dem108 can you help, with you microsoft connections to progress resolution of this issue please ?
Many thanks.  :)

**Comment by skasturi** [comment]
@rastala

**Comment by GiftA-MSFT** [comment]
@dem108 downgrading tensorflow to 1.14 resolved the issue for me. It seems support isn't available for 2.0 at the moment. 

@vizhur any idea on when support will be available (referencing [#637](https://github.com/Azure/MachineLearningNotebooks/issues/637))?

**Comment by dem108** [comment]
@GiftA-MSFT acknowledged with thanks!
